tabular/src/autogluon/tabular/trainer/model_presets/presets.py

```diff
@@ -104,6 +104,7 @@
     'valid_stacker',
     'valid_base',
     'hyperparameter_tune_kwargs',
+    'fit_with_prune'
```

Comment:
nit: This is a matter of preference, but I've found always ending list elements with a `,` even if its the last element saves a lot of pain over time.